string new_string
---
bool success